core-bundle/tests/OptIn/OptInTokenTest.php

```diff
@@ -306,10 +306,6 @@ public function testDoesNotRequireSubjectAndTextToResendToken(): void
 
     private function getToken(OptInModel $model, ContaoFramework $framework = null): OptInTokenInterface
     {
-        if (null === $framework) {
-            $framework = $this->mockContaoFramework();
-        }
-
-        return new OptInToken($model, $framework);
+        return new OptInToken($model, $framework ?? $this->mockContaoFramework());
```

Comment:
Same codestyle here:

```suggestion
	$framework ??= $this->mockContaoFramework();

        return new OptInToken($model, $framework);
```